notebooks/recommender-system.ipynb

```diff
@@ -0,0 +1,843 @@
+{
```

Comment:
<p>I hadn't used <code>nonzero</code>  before, another way is</p><p> </p><p><code>&gt;&gt;&gt; np.sum(ratings == 0)</code>  </p><p><code>90570</code>  </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>